index.html

```diff
@@ -0,0 +1,197 @@
+</head>
+
+<body>
+    <!--header section starts-->
+    <header>
+        <h2 class="logo"><a href="#">SP</a></h2>
+        <nav>
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#6070FF" />
+            </svg>
+        </nav>     
+    </header>
+    <!--header section ends-->
+
+    <!--headline section starts-->
+    <section class="headline-main">
+        <section class="headline">
+            <h2 class="headline-heading">I'm Suthish Prasanna<br>Glad to see you!</h2>
+            <p class="headline-paragraph">I’m a software developer! I can help you build a product , feature or website
+                Look
+                through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <p class="headline-sm-suport">LET’S CONNECT</p>
+            <ul class="social-icons">
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li> 
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            </ul>
+        </section>
+    </section>
+    <!--headline section ends-->
+
+<!--Card works 1 starts-->
+ <section class="works-card">
+    <img src="./assets/Snapshoot Portfolio.png" alt="Recent Work">
+    <h2 class="works-card-heading-large">Tonic</h2>
+    <div class="works-card-client">
+        <p class="paragraph">CANOPY</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-role paragraph">Back End Dev</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-year paragraph">2015</p>
+    </div>
+    <p class="paragraph works-card-paragraph">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="works-card-tags">
+        <li class="works-card-tag">html</li>
+        <li class="works-card-tag">css</li>
+        <li class="works-card-tag">javascript</li>
+    </ul>
+    <div>
+    <button>See Project</button>
+    </div>
+</section>
+<!--Card works 1 ends-->
+
+<!--Card works 2 starts-->
+<section class="works-card">
+    <img src="./assets/Snapshoot Portfolio (1).png" alt="Recent Work">
+    <h2 class="works-card-heading-large">Multi-Post<br>Stories</h2>
+    <div class="works-card-client">
```

Comment:
- [ ] While you did a fantastic job creating all your cards :+1:, I think it would be better if you wrap all the work cards inside one `section` tag because it is semantically correct :100: 

- [ ] Also, kindly set the background color of the section to #E5E5E5 instead of using #FFF to match with the Figma template

**Your design**

![Screenshot from 2023-02-16 08-05-14](https://user-images.githubusercontent.com/105117832/219372852-3bda10d1-7a4d-4f65-b9c7-5b381090fb27.png)

**Figma**

![Screenshot from 2023-02-16 08-05-38](https://user-images.githubusercontent.com/105117832/219372917-6477eacd-42a8-4e62-aa48-f29f0912beef.png)